src/app/AttributePathParams.h

```diff
@@ -65,11 +70,11 @@ struct AttributePathParams
         }
         return true;
     }
-    chip::NodeId mNodeId         = 0;
-    chip::EndpointId mEndpointId = 0;
-    chip::ClusterId mClusterId   = 0;
-    chip::FieldId mFieldId       = 0;
-    chip::ListIndex mListIndex   = 0;
+    NodeId mNodeId         = 0;
```

Comment:
Is the node id actually needed here, except for proxies?

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Encoding-Specification.adoc#path
I am doing one on one match between attributePath tlv representation and attributePathParams.

I think later nodeId would be removed? @mrjerryjohns

Reply:
I know why it's in there, sure.  I'm just saying most devices (esp. constrained ones) will not actually need it.  Followup tracking that is fine.